### PR TITLE
Virtual Rounding app template

### PR DESCRIPTION
**Pull request by adityachallapally**

### CODE REVIEW

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Please verify and update the comments.

File: msteams-platform/samples/app-templates.md [hunk 1]
```diff
@@ -559,6 +559,16 @@ Survey is a custom Microsoft Teams [messaging extension](../messaging-extensions
 :::column-end:::
 :::row-end:::
 
+## Virtual Rounding &#9734;
+
+Hospital and emergency room providers make dozens, and often hundreds of “rounds” per day. These quick check-ins on patients are intended to provide a status check on how the patient is doing and ensure that the patient’s concerns are addressed. While rounding is an essential practice to ensure patients are being monitored by multiple types of providers, they represent a huge drain on PPE, because for each visit, from each provider, a new mask, and new set of gloves must be used. With this app templates, medical workers can easily conduct rounds virtually, through a Microsoft Teams meeting between the provider and the patient.
+
+This is the Virtual Rounding solution referenced in the Microsoft Health and Life Sciences [blog post](https://aka.ms/teamsvirtualrounding).
```
Comment (surbhigupta12) [inline msteams-platform/samples/app-templates.md:566]: Suggest rephrasing this to:

```suggestion
The Virtual Rounding solution is also referenced in the Microsoft Health and Life Sciences [blog post](https://aka.ms/teamsvirtualrounding).
```

**Review by surbhigupta12 — COMMENTED** [review]
Minor comment, please update.

**Review by surbhigupta12 — APPROVED** [review]
Thank you for incorporating the comments. Approving the PR.

**Review by KirtiPereira — APPROVED** [review]
Approving the PR for merging.
Thank you.

**Review by surbhigupta12 — APPROVED** [review]
Changes seem fine. Approving the PR.

**Pull request merged**